src/m1_facilitator/repositories/TransactionRepository.ts

```diff
@@ -55,7 +55,7 @@ class TransactionModel extends Model {
 export default class TransactionRepository extends Subject<Transaction> {
   /* Public Functions */
 
-  public constructor(initOptions: InitOptions) {
+  public constructor(initOptions: InitOptions, modelName: string, tableName: string) {
```

Comment:
I am not sure if we should have separate transactions repository for origin/aux since the schema is same. Schema can easily be combined with metachain attribute. 
It's ok for now since I don't see any harm .